README.md

```diff
@@ -1,50 +1,69 @@
 # Flutter Maplibre GL
-[![Flutter CI](https://github.com/maplibre/flutter-maplibre-gl/actions/workflows/flutter_ci.yml/badge.svg)](https://github.com/maplibre/flutter-maplibre-gl/actions/workflows/flutter_ci.yml)
-[![Generate docs](https://github.com/maplibre/flutter-maplibre-gl/actions/workflows/generate_docs.yml/badge.svg)](https://github.com/maplibre/flutter-maplibre-gl/actions/workflows/generate_docs.yml)
```

Comment:
Should we keep these badges, additionally to the new ones?

Reply:
I removed them because both information is not really usefull for users when they view the package on pub.dev.
The Flutter CI workflow runs for commits made to pull requests too, right?

Additionally I'd like to remove the docs generation in the future because the links in the right side bar are broken: https://htmlpreview.github.io/?https://raw.githubusercontent.com/m0nac0/flutter-maplibre-gl/docs/doc/api/index.html. We can just use the hosted API docs from pub.dev: https://pub.dev/documentation/maplibre_gl/latest/